scripts/protocgen.sh

```diff
@@ -18,7 +18,7 @@ for dir in $proto_dirs; do
   buf protoc \
     -I "proto" \
     -I "third_party/proto" \
-    --gocosmos_out=plugins=interfacetype+grpc,\
```

Comment:
I can remove `interfacetype` here since it is no longer used in  [cosmos.proto](https://github.com/cosmos/cosmos-sdk/blob/cyberbono3/introduce-cosmos-scalars/third_party/proto/cosmos_proto/cosmos.proto), right?

Reply:
yes